### ISSUE TITLE
Autocomplete for dicts

### ISSUE THREAD

**Issue by iFrozenPhoenix** [tilt-dev/starlark-lsp#44]
Autocomplete for dicts would be very good. This would ease the configuration with structured data.

**Comment by nicksieger** [comment]
Can you write an example you'd like to see work?

**Comment by iFrozenPhoenix** [comment]
Sure
```python
var1 = {
  "metadata" = " aaa"
}
```

I'd like do access metadata if I write var1[.
Similar to the dot notation typescript provides for nested interfaces / maps.

**Comment by iFrozenPhoenix** [comment]
@nicksieger is it likely that this feature will be implemented in the near future.
It would really ease the configuration.

**Comment by nicksieger** [comment]
I agree it would be a nice-to-have. Without some support 👍 it would be hard to prioritize working on this soon though, thanks for understanding. If you want to attempt a PR yourself I would definitely take a look!
